content/20.treatments.md

```diff
@@ -302,8 +302,19 @@ All of these factors could potentially influence treatment outcome.
 Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
 
 **Summary.**
-In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
-Multiple clinical studies have already been carried out to assess this possibility.
+In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+It was recently discussed [@doi:10.1093/jac/dkaa114] that since EC90 of CQ in Vero E6 cells (6.90 μM) could be reached and well endured in rheumatoid arthritis patients, it could also be administered to COVID-19 patients.
+Additionally, it was proven to be effective against HIV [@doi:10.1016/0149-2918(95)80039-5] and chronic Hepatitis C infection [@doi:10.1002/jmv.24575].
+Therefore, initially it was thought that it would be effective towards SARS-COV-2.
+Further, HCQ was hypothesized to be safer as compared to CQ [@doi:10.1093/jac/dkaa114].
+The hypotheses were that HCQ could alleviate the development of COVID-19 through mitigation of CD154 in T cells,  could have the same effect as an antiviral like CQ pre- and post-infection, is cheaper, and has less side effects than CQ. 
```

Comment:
```suggestion
The hypothesis was that HCQ could alleviate the development of COVID-19 through the mitigation of CD154 in T cells and could function similar to an antiviral either pre- or post-infection.
HCQ received particular attention because it is cheaper to manufacture and has been hypothesized to be safer than CQ [@doi:10.1093/jac/dkaa114].
```
I'm not sure if this is what you were trying to say for the antiviral part? My understanding is that CQ and HCQ are extremely similar but CQ is produced from tree bark and HCQ is manufactured in the lab and has an additional hydroxyl group added.

Reply:
You may take these sentences out.